src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1,3 +1,6 @@
+imports:
+  - { resource: 'form_type/*' }
```

Comment:
@matks `bundle/form/form_type.yml` is huge already, so i've started spliting it, is that ok?

Reply:
Yes, since we're going to add almost x2 services for the whole migration